include/nbl/asset/utils/IVirtualTexture.h

```diff
@@ -1047,7 +1047,7 @@ class IVirtualTexture : public core::IReferenceCounted, public IVirtualTextureBa
         auto fillBinding = [](auto& bnd, uint32_t _binding, uint32_t _count, core::smart_refctd_ptr<sampler_t>* _samplers) {
             bnd.binding = _binding;
             bnd.count = _count;
-            bnd.stageFlags = asset::IShader::E_SHADER_STAGE::ESS_ALL;
+            bnd.stageFlags = asset::IShader::E_SHADER_STAGE::ESS_ALL_OR_LIBRARY;
```

Comment:
don't beat a dead horse, don't touch code we don't need ported, unless it throws compilation errors